### PR TITLE
MQE-2219: Part 6 Docs Annual Review

### PR DESCRIPTION
**Pull request by soumyau**
### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/section.md [hunk 1]
```diff
@@ -116,21 +116,21 @@ The most usual use case is a test step with a button click action.
 
 The section element code declaration containing the timeout attribute:
 
-> StorefrontSigninSection.xml
+> StorefrontCustomerSignInPopupFormSection.xml
 
 ```xml
 ...
-<element name="signIn" type="button" selector="#signIn" timeout="30"/>
+<element name="signIn" type="button" selector="#send2" timeout="30"/>
```
Comment (jilu1) [inline docs/section.md:123]: Is this change necessary?

File: docs/suite.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ You can form suites using separate tests, groups, and modules.
 
 Each suite must be defined in the `<VendorName>/<ModuleName>/Test/Mftf/Suite` directory.
 
-The tests for each suite are generated in a separate directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/`.
-All tests that are not within a suite are generated in the _default_ suite at `.../Magento/FunctionalTest/_generated/default/`.
+The tests for each suite are generated in a separate directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/_generated/`.
+All tests that are not within a suite are generated in the _default_ suite at `.../Magento/_generated/default/`.
```
Comment (jilu1) [inline docs/suite.md:10]: Let's match the path with what's in line 9.

File: docs/section.md (hunk 1 above)
Comment (soumyau) [reply docs/section.md:123]: updated it to match current value of `signIn` in `StorefrontCustomerSignInPopupFormSection`

File: docs/suite.md (hunk 2 above)
Comment (soumyau) [reply docs/suite.md]: done

Comment (jilu1) [reply docs/suite.md]: I mean change it to `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/_generated/default/`

Comment (soumyau) [reply docs/suite.md]: yes, that's what I did : )

File: docs/suite.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ You can form suites using separate tests, groups, and modules.
 
 Each suite must be defined in the `<VendorName>/<ModuleName>/Test/Mftf/Suite` directory.
 
-The tests for each suite are generated in a separate directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/`.
-All tests that are not within a suite are generated in the _default_ suite at `.../Magento/FunctionalTest/_generated/default/`.
+The tests for each suite are generated in a separate directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/_generated/`.
+All tests that are not within a suite are generated in the _default_ suite at `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/_generated/default`.
```
Comment (soumyau) [inline docs/suite.md:10]: @jilu1 is this path correct?

Comment (jilu1) [reply docs/suite.md:10]: This is correct
